Object_Geometry[] object_list
